src/Orchard.Cms.Web/Modules/Orchard.Resources/Assets.json

```diff
@@ -8,5 +8,9 @@
       "Assets/Bootstrap/Js/bootstrap.js"
     ],
     "output": "Content/Scripts/bootstrap.js"
+  },
+  {
+    "inputs": [ "Assets/font-awesome/scss/font-awesome.scss" ],
+    "output": "Content/Styles/font-awesome.css"
```

Comment:
In the end you can just copy the official css files in this folder. They shouldn't be generated by our pipeline.